common/autoware_motion_utils/src/marker/marker_helper.cpp

```diff
@@ -85,6 +85,42 @@ inline visualization_msgs::msg::MarkerArray createDeletedVirtualWallMarkerArray(
 
   return marker_array;
 }
+
+inline visualization_msgs::msg::MarkerArray createIntendedPassVirtualMarkerArray(
+  const geometry_msgs::msg::Pose & vehicle_front_pose, const std::string & module_name,
+  const std::string & ns_prefix, const rclcpp::Time & now, const int32_t id,
+  const std_msgs::msg::ColorRGBA & color)
+{
+  visualization_msgs::msg::MarkerArray marker_array;
+
+  // Virtual Wall
```

Comment:
```suggestion
```
not virtual wall

Reply:
fixed in 0dd5fe18ed43f624abcc90ca0a79de2752598f75